frontend/src/layouts/AppLayout/AppLayout.tsx

```diff
@@ -271,6 +280,30 @@ export const AppLayout = ({ children }: LayoutProps) => {
     }
   };
 
+  const addProjectToFavorites = useCallback(
```

Comment:
A bit confused by this usecallback?

Reply:
And the below one

Reply:
ah right this isn't necessary. will remove